src/main/java/com/tabnine/selections/TabNineLookupListener.java

```diff
@@ -22,9 +23,11 @@
 
 public class TabNineLookupListener implements LookupListener {
     private final BinaryRequestFacade binaryRequestFacade;
+    private final StatusBarUpdater statusBarUpdater;
 
     public TabNineLookupListener(BinaryRequestFacade binaryRequestFacade) {
         this.binaryRequestFacade = binaryRequestFacade;
+        this.statusBarUpdater = new StatusBarUpdater(binaryRequestFacade);
```

Comment:
get it in the ctor here as well. Initialize it in the `DependencyContainer` thingy